operator/src/lib.rs

```diff
@@ -131,42 +131,44 @@ impl NifiState {
         data.insert("nifi.properties".to_string(), nifi_properties);
         data.insert("state-management.xml".to_string(), state_management_xml);
 
+        // And now create the actual ConfigMap
+        let config_map = stackable_operator::config_map::create_config_map(
```

Comment:
We can leave it like this but check this out for the future:
https://doc.rust-lang.org/nightly/book/ch07-04-bringing-paths-into-scope-with-the-use-keyword.html#creating-idiomatic-use-paths